stylelint-polaris/index.js

```diff
@@ -0,0 +1 @@
+modules.exports = {};
```

Comment:
Absolutely 👍  I don't have any public facing rules to set right now, so I plan to leave it as an empty object (matching the proposed architecture). However, I think it would be great to follow the same structure as the [coverage plugin](https://github.com/Shopify/polaris/blob/f603d3e283770f41128f8713bf79f5c4a24693ee/polaris-react/stylelint/plugins/coverage/index.js#L6-L8) in the follow up consolidation tasks!